content/python/concepts/substrings/substrings.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: 'Substrings'
+Description: 'A substring is a sequence of characters that are part of an original string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A substring is a sequence of characters that are part of an original string. In Python, substrings can be obtained by using the slicing feature on a [string](https://www.codecademy.com/resources/docs/python/strings) variable. A slice can be made in a specific position within the string or it can be made at the default index.
+
+## Syntax
+
+A slice is made by using the open `[` and closed `]` brackets on a string variable. Inside the brackets, the position can be given.
+
+```psuedo
+string[start:end:step]
+```
+
+- `start` gives the initial position the slice will start from (default: 0)
+- `end` is the position where the slicing will end (default: -1)
+- `step` indicates the number of steps to take in between indexes (default: 1)
+
+## Examples
+
+The following examples are different ways to obtain subsets of a string. All examples are performed on the same string,
+
+```python
+string = "Codecademy"
+```
+
+### Retrieving Single Characters
+
+When only one index is specified, a single character is returned. An index of `0` retrieves the character at the beginning of the string:
+
+```python
+print(string[0])
+
+#Output: C
```

Comment:
```suggestion
# Output: C
```

Reply:
Let's apply this edit for the rest of the entry.

Reply:
Ditto on this 👆